packages/lib/src/vis/line/errorBarsGeometry.ts

```diff
@@ -1,62 +1,63 @@
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
We no longer have to disable this linting error in any of the geometry file because by getting rid of the `prepare` method, the `params` instance variable is now always defined.